src/latlon.py

```diff
@@ -12,14 +12,18 @@ def getlatlon(imgpath):
     im = rasterio.open(imgpath)
     crs = im.crs.__str__()  # Coordinate reference system code
     nrows, ncols = im.shape
-    rows, cols = np.meshgrid(np.arange(nrows), np.arange(ncols))
+    cols, rows = np.meshgrid(np.arange(ncols), np.arange(nrows))
```

Comment:
I guess this is doing the same as before? Why write this way?  Just curious.

Reply:
It's not the same as before, that's the point -- in the case where nrows == ncols, yes, it's just a change in names, but in general nrows != ncols. The output of mesh grid is two matrices that mark the indices, and for a square matrix the first matrix is the transpose of the other.